programs/mpl-core/src/processor/burn.rs

```diff
@@ -109,6 +121,12 @@ pub(crate) fn burn_collection<'a>(
     assert_signer(ctx.accounts.payer)?;
     let authority = resolve_authority(ctx.accounts.payer, ctx.accounts.authority)?;
 
+    if let Some(log_wrapper) = ctx.accounts.log_wrapper {
+        if log_wrapper.key != &spl_noop::ID {
+            return Err(MplCoreError::InvalidLogWrapperProgram.into());
+        }
+    }
+
```

Comment:
Good to have the check since the account is currently in the instruction, but I don't think we would ever use log_wrapper for burn_instruction given the instruction would not be compressed ever.  So maybe a different PR is to remove log_wrapper from this instruction but for now I agree with adding the check since the account is included.